scripts/migrate.js

```diff
@@ -0,0 +1,45 @@
+const { migrate } = require('postgres-migrations')
+const path = require('path')
+
+const { NODE_ENV } = process.env
+
+  // This code is repeated a few times in our scripts. Can you factor it out into a shared helper function?
```

Comment:
Its always worth proofreading your work before submitting it, as questions like this are there only to help you think about possible optimisations. Think of it from the perspective of a possible new employer, things like this  make it especially clear that this was pasted in from somewhere which doesn't leave a great impression

To be clear, this isnt a criticism since we said its okay to copy paste/use the create-backend-app script in this instance, just a good habit to get into